### PR TITLE
Adds Scroll rewards to various quests

### PR DESCRIPTION
**Pull request by Rfpnj**
### Things to include in a large Pull Request:

- What is the purpose of the pull request?
Adds the following Scroll awards to quests:
Nyan-Earth scroll
Brax-Fire scroll
Harpy quest-Air scroll
Arthur post package-Arcane scroll
- Give a brief description of what you changed or added.
Gives one scroll per quest
- Are any new graphical assets required?
no
- Has this change been tested? If so, mention the version number that the test was based on.
0.2.5.5
- So we have a better idea of who you are, what is your Discord Handle?

- If you want quick feedback, you can use @Innoxia, or jump on the Lilith's Throne discord and send Innoxia a PM.

### CODE REVIEW

File: src/com/lilithsthrone/game/dialogue/places/dominion/harpyNests/HarpyNestsDialogue.java [hunk 1]
```diff
@@ -445,7 +447,7 @@ public String getContent() {
 					+ "</p>"
 						+ "The muscular horse-boy produces a little brass key, and, bending down to unlock a heavy iron safe that's sitting beneath his desk, he retrieves a bag of money."
 						+ " Setting it down in front of him, he grins up at you."
-						+ " [style.speechMasculineStrong(I still can't believe that you actually managed to do it! Well, I'll keep my end of the bargain; here's five thousand flames!)]"
+						+ " [style.speechMasculineStrong(I still can't believe that you actually managed to do it! Well, I'll keep my end of the bargain; here's five thousand flames! Here have this scroll too, for the speed with whch you handled everything)]"
```
Comment (Pimvgd) [inline src/com/lilithsthrone/game/dialogue/places/dominion/harpyNests/HarpyNestsDialogue.java]: Missing sentence ending

File: src/com/lilithsthrone/game/dialogue/places/dominion/harpyNests/HarpyNestsDialogue.java [hunk 2]
```diff
@@ -446,7 +448,7 @@ public String getContent() {
 					+ "<p>"
 						+ "The muscular horse-boy produces a little brass key, and, bending down to unlock a heavy iron safe that's sitting beneath his desk, he retrieves a bag of money."
 						+ " Setting it down in front of him, he grins up at you."
-						+ " [style.speechMasculineStrong(I still can't believe that you actually managed to do it! Well, I'll keep my end of the bargain; here's five thousand flames!)]"
+						+ " [style.speechMasculineStrong(I still can't believe that you actually managed to do it! Well, I'll keep my end of the bargain; here's five thousand flames! Here have this scroll too, for the speed with whch you handled everything.)]"
```
Comment (CognitiveMist) [inline src/com/lilithsthrone/game/dialogue/places/dominion/harpyNests/HarpyNestsDialogue.java]: Spelling: whch

File: src/com/lilithsthrone/game/character/npc/dominion/Brax.java [hunk 3]
```diff
@@ -503,7 +509,8 @@ public String getContent() {
 					+ "</p>"
 					+ "<p>"
 						+ "Taking one last look around Brax's office, you notice that there's a neatly-folded spare uniform lying on top of a cabinet."
-						+ " Deciding to punish Brax a little more for trying to beat you up, <b>you take the spare uniform, and add it to your inventory</b>."
+						+ " It also has a scroll sitting on top of the pile of uniform pieces."
+						+ " Deciding to punish Brax a little more for trying to beat you up, <b>you take the spare uniform, and the scroll and add them to your inventory</b>."
```
Comment (CognitiveMist) [inline src/com/lilithsthrone/game/character/npc/dominion/Brax.java]: The comma should be moved from after 'uniform' to after 'scroll'.

File: src/com/lilithsthrone/game/character/npc/dominion/Brax.java [hunk 4]
```diff
@@ -503,7 +509,8 @@ public String getContent() {
 					+ "</p>"
 					+ "<p>"
 						+ "Taking one last look around Brax's office, you notice that there's a neatly-folded spare uniform lying on top of a cabinet."
-						+ " Deciding to punish Brax a little more for trying to beat you up, <b>you take the spare uniform, and add it to your inventory</b>."
+						+ " It also has a scroll sitting on top of the pile of uniform pieces."
```
Comment (CognitiveMist) [inline src/com/lilithsthrone/game/character/npc/dominion/Brax.java]: This sentence feels clunky to me. Maybe something like "...there's a neatly-folded spare uniform lying on top of a cabinet, next to a small scroll.

**Pull request closed without merging**